File: libs/common-lib/src/main/java/com/coinhub/common/util/StringUtils.java
package com.coinhub.common.util;

public final class StringUtils {

    private StringUtils() {
    }

    public static boolean isBlank(String str) {
        return str == null || str.trim().isEmpty();
    }

    public static boolean isNotBlank(String str) {
        return !isBlank(str);
    }

    public static String trimToEmpty(String str) {
        return str == null ? "" : str.trim();
    }
}

File: libs/dotnet-lib/StringUtils.cs
namespace CoinHub.Common;

public static class StringUtils
{
    public static bool IsBlank(string? str) =>
        string.IsNullOrWhiteSpace(str);

    public static bool IsNotBlank(string? str) =>
        !IsBlank(str);

    public static string TrimToEmpty(string? str) =>
        str?.Trim() ?? string.Empty;
}
